README.md

```diff
@@ -114,6 +114,15 @@ The GP2GP uses the [MHS Adaptor]() to send/receive messages to/from Spine.
 | -------------------------------------|-----------------------------------------------|-------------
 | GP2GP_MHS_OUTBOUND_URL               | http://localhost:8081/mock-mhs-endpoint       | URL to the MHS adaptor's outbound endpoint
 
+### Trust Store Configuration Options
+
+The adaptor may be assigned with a predefined trustore configuration for Amazon S3.
```

Comment:
Please use this text adaptor from NHAIS readme

You can configure a trust store with private CA certificates if required for TLS connections. The trust store does not replace Java's default trust store. At runtime the application adds these additional certificates to the default trust store. Only an s3:// url is currently supported, and the current use-case is to support AWS DocumentDb.